### ISSUE TITLE
Issue with generating lmdb

### ISSUE THREAD

**Issue by Princesston** [facebookresearch/vilbert-multi-task#11]
Running the `script/convert_to_lmdb.py` stops at around 90% with no error, but apparently it hasn't finished, as running `train_tasks.py` gives `TypeError: a bytes-like object is required, not 'NoneType'` at [here](https://github.com/facebookresearch/vilbert-multi-task/blob/72770f18688bb55e0a0eb5aaf389055eb3acdd6e/vilbert/datasets/_image_features_reader.py#L59). Does anyone know what could be the problem? Thanks~

**Comment by jiasenlu** [comment]
yeah, it will have a problem. Since we encode the mapping dict in the last line of covert_to_lmdb.py script. Could you try the script again? or convert with fewer images?

**Comment by vedanuj** [comment]
All the data for the downstream tasks can be downloaded now: Please check this update #26  

Readme here : https://github.com/facebookresearch/vilbert-multi-task/tree/master/data
